pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -326,7 +326,7 @@ func populateVsphereInstanceMap(cfg *VSphereConfig) (map[string]*VSphereInstance
 				Username:          vcConfig.User,
 				Password:          vcConfig.Password,
 				Hostname:          vcServer,
-				Insecure:          vcConfig.InsecureFlag,
+				Insecure:          cfg.Global.InsecureFlag,
```

Comment:
I think you don't need 
line 88: InsecureFlag bool `gcfg:"insecure-flag"` in VirtualCenterConfig

Reply:
Yes, we don't this now. Will remove this

Reply:
Isn't InsecureFlag specific to how we want to connect to VC. The user may want to connect insecure in to some VCs and secure to other VCs. Am I missing something?

Reply:
Since InsecureFlag is bool we can't determine why InsecureFalg is false.
If user doesn't specify InsecureFlag in vCenter then boolean value gets false as default value and then it can't be overriden with global config (and global can be true).

So we thought only in Global user can specify InsecureFlag and it will be applicable to all VCs.

Other solution would be to use string instead of boolean.
InsecureFalg = "Yes" / "No"